third_party/cargo/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+name = "compile_with_bazel"
+version = "0.0.0"
+
+# Mandatory (or Cargo tooling is unhappy)
+[lib]
+path = "fake_lib.rs"
+
+[dependencies]
+# librubyfmt's dependencies
+serde = { version = "1.0", features=["derive"] }
+serde_json = "1.0.40"
+bytecount = "0.6.0"
+backtrace = "0.3.45"
+libc = "0.2.68"
+log = { version = "0.4.8", features = ["max_level_debug", "release_max_level_warn"] }
+simplelog = "0.8"
+# ripper deserialize's dependencies
+syn = { version = "=1.0.17", features = ["full"] }
+quote = "=1.0.3"
+proc-macro2 = "=1.0.10"
+[build-dependencies]
+cc = "1.0"
```

Comment:
Are these dependencies duplicated out of rubyfmt's source tree? How will we know if these are out of date / how to update them?

Reply:
They are. This is the most fragile part, so I'll write a readme for this.